packages/angstrom-lwt-unix/angstrom-lwt-unix.0.16.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: [ "Spiros Eliopoulos <spiros@inhabitedtype.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/angstrom"
+bug-reports: "https://github.com/inhabitedtype/angstrom/issues"
+dev-repo: "git+https://github.com/inhabitedtype/angstrom.git"
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```
`pinned` for `subst` is not correct and is a bug fixed in [dune.2.7.0](https://github.com/ocaml/dune/blob/main/CHANGES.md#270-13082020)